src/content/content.service.ts

```diff
@@ -0,0 +1,92 @@
+import { Injectable, Inject } from '@nestjs/common';
+import { GitDBService } from '../git-db/git-db.service';
+import { SetCallback } from '@flatlify/gitdb';
+
+interface getManyParams {
+  pagination?: {
+    perPage?: number;
```

Comment:
offset and limit params are more flexible to operate for pagination. looks like it should be straight forward to update on the FE side